graphql_schema.json

```diff
@@ -3287,7 +3574,7 @@
             },
             {
               "name": "pooledUserCommands",
-              "description": "Retrieve all the user commands sent by a public key",
+              "description": "Retrieve all the user commands submitted by the current daemon that are pending inclusion",
```

Comment:
eventually would like to start calling these `user transactions`

Reply:
In general there are some naming decisions in the graphql that will probably need to be made, but I'm trying to avoid making them piecemeal.
wallet -> account
user command -> user txn
etc